python/sklearnserver/README.md

```diff
@@ -36,3 +36,71 @@ __main__.py: error: the following arguments are required: --model_dir
+Requirement already satisfied: attrs>=17.4.0 in /Library/Frameworks/Python.framework/Versions/3.7/lib/python3.7/site-packages (from pytest->sklearnserver==0.1.0) (19.1.0)
+Requirement already satisfied: setuptools in /Library/Frameworks/Python.framework/Versions/3.7/lib/python3.7/site-packages (frompytest->sklearnserver==0.1.0) (40.8.0)
+Requirement already satisfied: wcwidth in /Library/Frameworks/Python.framework/Versions/3.7/lib/python3.7/site-packages (from pytest->sklearnserver==0.1.0) (0.1.7)
+Requirement already satisfied: atomicwrites>=1.0 in /Library/Frameworks/Python.framework/Versions/3.7/lib/python3.7/site-packages (from pytest->sklearnserver==0.1.0) (1.3.0)
+Requirement already satisfied: pluggy!=0.10,<1.0,>=0.9 in /Library/Frameworks/Python.framework/Versions/3.7/lib/python3.7/site-packages (from pytest->sklearnserver==0.1.0) (0.11.0)
+Requirement already satisfied: more-itertools>=4.0.0; python_version > "2.7" in /Library/Frameworks/Python.framework/Versions/3.7/lib/python3.7/site-packages (from pytest->sklearnserver==0.1.0) (7.0.0)
+Requirement already satisfied: six>=1.10.0 in /Library/Frameworks/Python.framework/Versions/3.7/lib/python3.7/site-packages (from pytest->sklearnserver==0.1.0) (1.12.0)
+Requirement already satisfied: py>=1.5.0 in /Library/Frameworks/Python.framework/Versions/3.7/lib/python3.7/site-packages (from pytest->sklearnserver==0.1.0) (1.8.0)
+Requirement already satisfied: typed-ast<1.4.0,>=1.3.1 in /Library/Frameworks/Python.framework/Versions/3.7/lib/python3.7/site-packages (from mypy->sklearnserver==0.1.0) (1.3.5)
+Requirement already satisfied: mypy-extensions<0.5.0,>=0.4.0 in /Library/Frameworks/Python.framework/Versions/3.7/lib/python3.7/site-packages (from mypy->sklearnserver==0.1.0) (0.4.1)
+Installing collected packages: sklearnserver
+  Found existing installation: sklearnserver 0.1.0
+    Uninstalling sklearnserver-0.1.0:
+      Successfully uninstalled sklearnserver-0.1.0
+  Running setup.py develop for sklearnserver
+Successfully installed sklearnserver
+```
+
+To run tests, please change the test file to point to your model.joblib file. Then run the following command:
+
+```bash
+make test
+```
+
+The following shows the type of output you should see:
+
+```
+pytest -W ignore
+====================================================== test session starts ======================================================
+platform darwin -- Python 3.7.3, pytest-4.5.0, py-1.8.0, pluggy-0.11.0
+rootdir: /Users/animeshsingh/DevAdv/kfserving/python/sklearnserver
+plugins: tornasync-0.6.0.post1
+collected 1 item
+
+sklearnserver/test_model.py .                                                                                             [100%]
+
+=================================================== 1 passed in 0.43 seconds ====================================================
+```
+
+To run static type checks:
+
+```bash
+mypy --ignore-missing-imports xgbserver
```

Comment:
Can we put this in the makefile?. Also xgbserver -> sklearnserver

Reply:
xgbserver -> sklearnserver - Done.

The test is in makefile. I can remove the description here if you think so

Reply:
Oh I was suggesting we just run it as part of the test flow. Kind of like a "vet"

Reply:
Done